### PR TITLE
[zh-cn]SYNC feature-gates.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -494,7 +494,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `APIResponseCompression`：压缩 “LIST” 或 “GET” 请求的 API 响应。
 - `APISelfSubjectReview`：激活 `SelfSubjectReview` API，允许用户查看请求主体的身份验证信息。
   更多细节请参阅 [API 访问客户端的身份验证信息](/zh-cn/docs/reference/access-authn-authz/authentication/#self-subject-review)。
-- `APIServerIdentity`：为集群中的每个 API 服务器赋予一个 ID。
+- `APIServerIdentity`：为集群中的每个 API 服务器赋予一个 ID，使用[租约](/zh-cn/docs/concepts/architecture/leases)。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:497]: 注意中文語序：

```suggestion
- `APIServerIdentity`：使用[租约](/zh-cn/docs/concepts/architecture/leases)为集群中的每个
  API 服务器赋予一个 ID。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -494,7 +494,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `APIResponseCompression`：压缩 “LIST” 或 “GET” 请求的 API 响应。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:494]:
```suggestion
- `APIResponseCompression`：压缩 `LIST` 或 `GET` 请求的 API 响应。
```
Line 492 has the same issue.

**Review by windsonsea — COMMENTED** [review]
As a reviewer, it's better to review details of what you have involved in, to show your knowledge there. This is only a suggestion.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**